src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackV2VolumePluginTest.java

```diff
@@ -162,16 +187,22 @@ public void removeInstance() throws IOException {
         this.expectedRequest = new HttpUriRequestMatcher(request, null);
 
         try {
+            // exercise
             this.openStackV2VolumePlugin.deleteInstance(FAKE_INSTANCE_ID, this.tokenDefault);
         } catch (Exception e) {
         }
 
+        // verify
         Mockito.verify(client).execute(Mockito.argThat(this.expectedRequest));
     }
 
+    // test case: Deleting an instance without a tenant ID must raise FogbowManagerException.
     @Test(expected=FogbowManagerException.class)
     public void testRemoveInstanceWithoutTenantId() throws Exception {
+        // set up
         this.tokenDefault.getAttributes().clear();
+
+        // exercise
         this.openStackV2VolumePlugin.deleteInstance(FAKE_INSTANCE_ID, tokenDefault);
     }
 
```

Comment:
why do we have this empty ignored test case in line 211? Do you want to indicate it should be implemented in the future? If so, remove the code, keep the //TODO

Reply:
Whoever did this class meant it, indeed.